deploy/images/controller/Dockerfile

```diff
@@ -1,17 +1,20 @@
-# Use distroless image which already includes ca-certificates
-FROM gcr.io/distroless/static:nonroot
+FROM ubuntu:latest
 
 # Argument for target architecture
 ARG TARGETARCH
 
 # Set the working directory
 WORKDIR /
 
+# Install the required dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        libicu-dev \
+        ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+
 # Copy the application binary for the specified architecture
 COPY ./linux_${TARGETARCH:-amd64}/release/controller /
 
-# Set the user to non-root (65532:65532 is the default non-root user in distroless)
```

Comment:
Was running as the non-root user causing issues?

Reply:
This is specific to distroless. I'll figure out what we should do for alpine and ubuntu

Reply:
fixed - controller now runs as non-root